BoxCollider.hpp

```diff
@@ -12,7 +12,7 @@ namespace spic {
     class BoxCollider : public Collider {
         public:
 
-            BoxCollider(double width, double height, std::vector<int> collideLayers, Vector2 offset) : Collider(collideLayers, offset);
+            BoxCollider(double width, double height, std::vector<int> collideLayers, Vector2<float> offset) : Collider(collideLayers, offset);
```

Comment:
Wat is de reden dat de vector float is en niet een double?

Reply:
Scherp, dit moet idd een double zijn.